### ISSUE TITLE
Where to translate?

### ISSUE THREAD

**Issue by p-bo** [FreeRDP/Remmina#1462]
Hello,

is still https://translations.launchpad.net/remmina place where to translate?
.po files on github and launchpad are out of sync.
It isn't even mentioned on https://www.remmina.org/wp/contributing-to-remmina/

Thanks :)

**Comment by antenore** [comment]
We don't know how to sync the translations on launchpad. The github ones are the reference.
Do you know how to sync both magically?

**Comment by p-bo** [comment]
From launchpad it isn't possible unfortunately.
But if you migrate to weblate, it is:
https://docs.weblate.org/en/latest/admin/continuous.html
It is free (libre) software and for free/opensource project there is even costless offer for hosting:
https://weblate.org/en/hosting/
See: Hosting for free software

Advantage of web based translation tool is lowering bar for translators - if you want even not so technical people to contribute in this area (as with they there is better chance they are able to express things in language more familiar to end users), operating with source code version control system, which is natural for developers, would be quite discouraging for non-technical people.

**Comment by antenore** [comment]
I agree on everything. We will discuss about this and we will take a look, thanks!

@giox069

**Comment by der-eismann** [comment]
Solus also uses transifex, which is similar I guess. Also free for open source projects:
https://www.transifex.com/pricing/

**Comment by p-bo** [comment]
Note that Transifex is proprietary software and service which terms of use and behavior may change in future - do remember once popular Source Forge for instance - it is understandable, that project that is already using Transifex will not move due it, but for new deployment it is important to think about sustainability.
